.env

```diff
@@ -1 +1,25 @@
-DATABASE_URL=postgres://postgres:postgres@localhost/fang
+POSTGRES_CONTAINER=postgres
```

Comment:
maybe it's worth setting the version of the postgres, mysql and sqlite

Reply:
how do you specify the version ?

Reply:
Sounds like a good idea. I will look into it.

Reply:
@Dopplerian can we use just the current latest version and update it manually instead of always using the latest tag?

because new versions of dbs can introduce breaking changes